qdrant-landing/content/documentation/frameworks/mem0.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Mem0
+---
+
+# Mem0
+
+[Mem0](https://mem0.ai) is a self-improving memory layer for LLM applications, enabling personalized AI experiences that save costs and delight users. Mem0 remembers user preferences, adapts to individual needs, and continuously improves over time, ideal for chatbots and AI systems.
+
+Mem0 supports various vector store providers, including Qdrant, for efficient data handling and search capabilities.
+
+## Installation
+
+To install Mem0 with Qdrant support, use the following command:
+
+```sh
+pip install mem0ai
+```
+
+## Usage
+
+Here's a basic example of how to use Mem0 with Qdrant:
+
+```python
+import os
+from mem0 import Memory
+
+os.environ["OPENAI_API_KEY"] = "sk-xx"
+
+config = {
+    "vector_store": {
+        "provider": "qdrant",
+        "config": {
+            "collection_name": "test",
+            "host": "localhost",
+            "port": 6333,
+        }
+    }
+}
+
+m = Memory.from_config(config)
+m.add("Likes to play cricket on weekends", user_id="alice", metadata={"category": "hobbies"})
+```
+
+## Configuration
+
+When configuring Mem0 to use `Qdrant` as the vector store, you can specify various parameters in the `config` dictionary. Here are the available options:
+
+| Parameter | Description | Default Value |
+|-----------|-------------|---------------|
+| `collection_name` | The name of the collection to store the vectors | `mem0` |
+| `embedding_model_dims` | Dimensions of the embedding model | `1536` |
+| `client` | Custom client for Qdrant | `None` |
+| `host` | The host where the Qdrant server is running | `None` |
+| `port` | The port where the Qdrant server is running | `None` |
+| `path` | Path for the Qdrant database | `/tmp/qdrant` |
+| `url` | Full URL for the Qdrant server | `None` |
+| `api_key` | API key for the Qdrant server | `None` |
+| `on_disk` | For enabling persistent storage | `False` |
```

Comment:
```suggestion
When configuring Mem0 to use Qdrant as the vector store, you can specify [various parameters](https://docs.mem0.ai/components/vectordbs/dbs/qdrant#config) in the `config` dictionary.
```

Reply:
Would be one less place to update if there are any changes.

Reply:
Yes much better. Thanks!